src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/pockets.json

```
@@ -7,7 +7,7 @@
   ],
   "effects": [
     "增加更多物品栏槽位",
-    "潜行状态下按“护腿交互”键打开护腿所属物品栏",
+    "同时按下Shift键与“护腿交互”键打开护腿所属物品栏",
```

```suggestion
    "同时按下左Shift键与“护腿交互”键打开护腿所属物品栏",
```
Depends on whether the Left Shift is specified.